packages/web-app-user-management/src/components/Users/SideBar/EditPanel.vue

```diff
@@ -1,39 +1,53 @@
 <template>
-  <div v-if="user" class="oc-mt-xl">
+  <div class="oc-mt-xl">
     <UserInfoBox :user="user" />
-    <div v-if="editUser" class="oc-background-highlight oc-p-m">
-      <oc-text-input
-        v-model="editUser.displayName"
-        class="oc-mb-s"
-        :label="$gettext('First and last name')"
-        :error-message="formData.displayName.errorMessage"
-        :fix-message-line="true"
-        @input="validateDisplayName"
-      />
-      <oc-text-input
-        v-model="editUser.mail"
-        class="oc-mb-s"
-        :label="$gettext('Email')"
-        :error-message="formData.email.errorMessage"
-        type="email"
-        :fix-message-line="true"
-        @change="validateEmail"
-      />
-      <oc-text-input
-        v-model="editUser.passwordProfile.password"
-        class="oc-mb-s"
-        :label="$gettext('Password')"
-        type="password"
-        :fix-message-line="true"
-        default-value="●●●●●●●●"
-      />
-      <oc-select
-        v-model="editUser.role"
-        :label="$gettext('Role')"
-        option-label="displayName"
-        :options="roles"
-        :clearable="false"
-      />
+    <div class="oc-background-highlight oc-p-m">
+      <div>
+        <oc-text-input
+          v-model="editUser.displayName"
+          class="oc-mb-s"
+          :label="$gettext('First and last name')"
+          :error-message="formData.displayName.errorMessage"
+          :fix-message-line="true"
+          @input="validateDisplayName"
+        />
+        <oc-text-input
+          v-model="editUser.mail"
+          class="oc-mb-s"
+          :label="$gettext('Email')"
+          :error-message="formData.email.errorMessage"
+          type="email"
+          :fix-message-line="true"
+          @change="validateEmail"
+        />
+        <oc-text-input
+          v-model="editUser.passwordProfile.password"
+          class="oc-mb-s"
+          :label="$gettext('Password')"
+          type="password"
+          :fix-message-line="true"
+          default-value="●●●●●●●●"
+        />
+        <div class="oc-mb-s">
+          <oc-select
+            v-model="editUser.role"
+            :label="$gettext('Role')"
+            option-label="displayName"
+            :options="roles"
+            :clearable="false"
+          />
+          <div class="oc-text-input-message"></div>
+        </div>
+        <quota-select
+          v-if="showQuota"
+          :title="$gettext('Personal quota')"
+          :total-quota="editUser.drive.quota.total || 0"
+          @selectedOptionChange="changeSelectedQuotaOption"
+        />
+        <div v-else v-translate>
```

Comment:
Am I still allowed to say that this should ideally be a `<p>` tag instead of a `<div>`? 🙈 sorry that I didn't see that before...